qa/common/deploy.sh

```diff
@@ -38,7 +38,6 @@ function _set_deepsea_minions {
 }
 
 function _initialize_minion_array {
-    set +x
```

Comment:
Not sure how this will improve the logging.

```
salt-key -L -l acc | grep -v ...
```

fails, it does not show nothing, and since '-e' is enabled, it will exit from the script.

Reply:
Oh, it will log which command failed.
Then looks good for me, but initial intention was to hide keys, right?